01-messaging.md

```diff
@@ -221,8 +221,11 @@ Various fundamental types are referred to in the message specifications:
 
 * `byte`: an 8-bit byte
 * `u16`: a 2 byte unsigned integer
+* `i16`: a 2 byte signed integer
 * `u32`: a 4 byte unsigned integer
+* `i32`: a 4 byte signed integer
 * `u64`: an 8 byte unsigned integer
+* `i64`: an 8 byte signed integer
```

Comment:
Do we need to specify the signed representation (i.e. two's complement)?

Reply:
I hesitated, so I decided to be lazy :laughing: 
I couldn't find an RFC or any stable "official" document to link to, and didn't want to specify it from scratch here...
What do you think we should do? Maybe @rustyrussell would know where to link readers to?

Reply:
As discussed during yesterday's spec meeting, I didn't link to any external documentation but explicitly stated that we're using big-endian two's complement and included test vectors for it.